src/ontology/mondo-edit.obo

```diff
@@ -445047,6 +445044,18 @@ xref: UMLS:C2718017 {source="MONDO:equivalentTo"}
 is_a: MONDO:0021179 {source="MESH:D057177", source="UMLS:C2718017"} ! proteostasis deficiencies
 created_by: http://orcid.org/0000-0002-4142-7153
 
+[Term]
+id: MONDO:0700041
+name: neuroblastoma, susceptibility to, 2
+synonym: "NBLST2" EXACT [] {source="OMIM:613013", synonymtypedef="ABBREVIATION"}
```

Comment:
the abbreviation annotation is not correct - you should select has_synonym_type, then Entity IRI/Annotation Properties, abbreviation (child of synonym_type property)

I fixed it